<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>GAME</string>
  </property>
  <property name="windowIcon">
   <iconset resource="testres/testres.qrc">
    <normaloff>:/test/nku.png</normaloff>:/test/nku.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="PlayBtn">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
         <property name="autoRepeatInterval">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="EditBtn">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OptionBtn">
         <property name="text">
          <string>Option</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExitBtn">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="tutorial"/>
    <addaction name="protutorial"/>
    <addaction name="potentialtable"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>寻找组织</string>
    </property>
    <addaction name="information"/>
    <addaction name="group"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="tutorial">
   <property name="text">
    <string>新手教程</string>
   </property>
  </action>
  <action name="protutorial">
   <property name="text">
    <string>进阶教程</string>
   </property>
  </action>
  <action name="information">
   <property name="text">
    <string>作者信息</string>
   </property>
  </action>
  <action name="group">
   <property name="text">
    <string>交流群</string>
   </property>
  </action>
  <action name="potentialtable">
   <property name="text">
    <string>定数表查询</string>
   </property>
   <property name="toolTip">
    <string>定数表查询</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="testres/testres.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ExitBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
